src/features/Chart/ChartManager.ts

```diff
@@ -1,6 +1,94 @@
-import { DatabaseService } from '../../common/services';
+import { DatabaseService } from '~common/services';
+import { ById } from '~common/types/common';
+import { ChartDataPoint, ChartTimestamp } from '~features/Chart/types';
 
-const GET_CHART_DATA = `
+enum ChartDataQueryKey {
```

Comment:
Query changes are to get all the chart data in a single query, and also to get the chart data for one sensor in one query (previously was doing a query for the timestamps in a separate query)